packages/i18n-utils/src/localize-url.tsx

```diff
@@ -129,6 +129,7 @@ export const urlLocalizationMapping: UrlLocalizationMapping = {
 	'wordpress.com/wp-admin/': setLocalizedUrlHost( 'wordpress.com', magnificentNonEnLocales ),
 	'wordpress.com/wp-login.php': setLocalizedUrlHost( 'wordpress.com', magnificentNonEnLocales ),
 	'jetpack.com': prefixLocalizedUrlPath( jetpackComLocales ),
+	'cloud.jetpack.com': prefixLocalizedUrlPath( jetpackComLocales ),
```

Comment:
Can you create automated tests - [similar to what is done for Jetpack](https://github.com/Automattic/wp-calypso/blob/36995755c35c242fb29f630282072d96590e9081/packages/i18n-utils/src/test/localize-url.js#L451) - to verify that any changes to this behaviour are caught in the future? 

For consistency, we'll have to update fbhepr%2Skers%2Sjcpbz%2S.pbasvt%2Sybpnyrf.cuc%3Se%3Qp0571s0p%231513-og

Reply:
Indeed, will do.

> For consistency, we'll have to update

Something was lost in translation here... ;^)

Reply:
Added tests here: d335f76

Reply:
> For consistency, we'll have to update
I'm wondering by looking at this if it means that we need to update `localize_url` to look for other locale subdomain options (ideally like nl.cloud.jetpack.com/manage/pricing), and makes sure to convert them - if I understand correctly?

Reply:
An update is not required as subdomains for cloud.jetpack.com do not exist.

Reply:
Circling back, updated here as well: D137788-code